experimenter_driver.py

```diff
@@ -112,13 +113,19 @@ def run(self):
             else:
                 pipes = self.get_pipelines_from_path(self.pipeline_location)
                 experimenter = Experimenter(self.datasets_dir, self.volumes_dir,
-                                            generate_pipelines=False, generate_problems=True)
+                                            generate_pipelines=False, generate_problems=True,
+                                            pipeline_gen_type=args.pipeline_gen_type,
+                                            n_classifiers=args.n_classifiers,
+                                            n_preprocessors=args.n_preprocessors)
```

Comment:
These extra arguments are needed because the `Experimenter` is no longer being passed the CLI arguments object as a whole.